docs/resources/elb_active_standby_pool.md

```diff
@@ -79,10 +78,9 @@ The following arguments are supported:
 
   -> **NOTE:** At least one of `loadbalancer_id`, `listener_id`, `type` must be specified.
 
-* `name` - (Optional, String, ForceNew) Specifies the backend server group name.
+* `name` - (Optional, String, ForceNew) Specifies the name of the backend server group.
```

Comment:
```suggestion
* `name` - (Optional, String, ForceNew) Specifies the name of the pool.
```
the resource is named pool, there should not be anything any thing about backend server group''
so please check other parameters and refer to the doc of resource elb_pool to make the description better

Reply:
done